src/devices/OpenHardwareMonitor/OpenHardwareMonitor.cs

```diff
@@ -0,0 +1,740 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
```

Comment:
should Windows implementation of CpuTemperature use this class instead of doing queries directly?

Reply:
I considered this as well, but since this class requires an external application to be running, I tought that it might be good to not add that dependency to the basic CpuTemperature class. But since the current implementation of CpuTemperature often doesn't work (depending on hardware), it may be better to change it anyway. 

So we have a solution (current implementation of CpuTemperature) that often doesn't work and needs elevation vs a solution that requires external components. Which do you prefer?

Reply:
I think whichever one is more useful. We should start with making the code work with good APIs, next steps are perf and size

Reply:
I'll try to use the OpenHardwareMonitor implementation on Windows now, and use the direct approach only as fallback.